draft-ietf-tls-tls13.md

```diff
@@ -2746,13 +2711,18 @@ from the resumption master secret:
                             "resumption context", "", Hash.Length)
 ~~~~
 
-The client MAY use this PSK for future handshakes by including
-the ticket value in the "pre_shared_key" extension in its ClientHello
-({{pre-shared-key-extension}}) and supplying a suitable PSK cipher
-suite. Servers may send multiple tickets on a single connection, for
-instance after post-handshake authentication. For handshakes that
-do not use a resumption_psk, the resumption_context is a string of
-Hash.Length zeroes.
+The client MAY use this PSK for future handshakes by including the
+ticket value in the "pre_shared_key" extension in its ClientHello
+({{pre-shared-key-extension}}). Servers may send multiple tickets on a
```

Comment:
@grittygrease the text about "after post-handshake auth" is referring to how the state the server wants to keep in the ticket changes when the client does post-handshake authentication, so the server probably wants to issue new tickets with that information.  The tickets are not used during post-handshake auth, but there is a reason to regenerate tickets after post-handshake auth.